Makefile

```diff
@@ -41,12 +41,12 @@ setup/git/hooks:
 
 .PHONY: code/run
 code/run:
-	@operator-sdk up local --namespace=$(NAMESPACE)
+	@export CR_NAMESPACE=${NAMESPACE}; operator-sdk up local --namespace=""
 
 .PHONY: code/run/service_account
 code/run/service_account: setup/service_account
 	@oc login --token=$(shell oc serviceaccounts get-token integreatly-operator -n ${NAMESPACE})
-	@operator-sdk up local --namespace=$(NAMESPACE)
+	@export CR_NAMESPACE=${NAMESPACE}; operator-sdk up local --namespace=""
```

Comment:
can make directives call make on the same Makefile they're in?

Reply:
Yes, although I think it should be `$(MAKE) code/run` rather than what I said before.

https://www.gnu.org/software/make/manual/make.html#MAKE-Variable